### ISSUE TITLE
[New Rule] Duplicate blocks in if/else

### ISSUE THREAD

**Issue by sanxiyn** [MartinThoma/flake8-simplify#192]
## Explanation

Inspired by `if_same_then_else` rule in Rust Clippy. See https://rust-lang.github.io/rust-clippy/master/index.html#if_same_then_else. SIM114(#10) is similar, but it only checks if/elif, not if/else.

## Example

```python
# Bad
if a:
    b
else:
    b

# Good
b
```
